### PR TITLE
Added Mineclone 2 compatibility

### PR DESCRIPTION
**Pull request by wuniversales**

### CODE REVIEW

File: brazier.lua [hunk 1]
```diff
@@ -1,3 +1,16 @@
+-- check for MineClone2
+local mcl = minetest.get_modpath("mcl_core")
+
+if mcl then
+    item1='mcl_core:torch'
```
Comment (ExcaliburZero) [inline brazier.lua]: Can you change the names of the variables `item1`, `item2`, and `item3` to `torch`, `cobble`, and `basic_flame` respectively (in each of the lua files)? That can help to make the source code for the recipes easier to understand at a glance.

**Review by ExcaliburZero — CHANGES_REQUESTED** [review]
Looks good, just one small change requested.

File: brazier.lua (hunk 1 above)
Comment (wuniversales) [reply brazier.lua]: Updated :)

Comment (ExcaliburZero) [reply brazier.lua]: Looks good.

**Review by ExcaliburZero — APPROVED** [review]

**Pull request merged**